lib/engine-addon.js

```diff
@@ -406,21 +476,15 @@ module.exports = {
         // LAZY LOADING!
         // But we have to implement everything manually for the lazy loading scenario.
 
-        // Get base styles tree.
-        var engineStylesTree = buildEngineCSSTree.call(this);
+        var vendorTree = buildVendorTree.call(this);
+        var vendorJSTree = buildVendorJSTree.call(this, vendorTree);
+        var externalTree = new Funnel(vendorTree, {
+          srcDir: 'vendor',
+          destDir: 'vendor',
+          allowEmpty: true
+        });
 
-        // Move styles tree into the correct place.
-        // `**/*.css` all gets merged.
-        // The addon.css file has already been renamed to match `this.name`.
-        // All we need to do is concatenate it down.
-        var primaryStyleTree;
-        if (engineStylesTree) {
-          primaryStyleTree = concat(engineStylesTree, {
-            allowNone: true,
-            inputFiles: ['**/*.css'],
-            outputFile: 'engines-dist/' + this.name + '/assets/engine.css'
-          });
-        }
+        var finalStylesTree = this.compileLazyEngineStyles(vendorTree, externalTree);
```

Comment:
Added that fixture and rebased, https://travis-ci.org/ember-engines/ember-engines/jobs/192898083 shows no diff before and after this change.  

Also, if you are curious, the assets do seem to be appended to the correct place `dist/lazy-in-lazy/assets/engine-vendor.css`.